src/view/abstracttexteditor.cpp

```diff
@@ -1062,6 +1066,109 @@ void AbstractTextEditor::onKeyboardStateChanged(QString state) {
     d->keyboardState = state;
 }
 
+void AbstractTextEditor::sendKeySequence(const QString &action, const QKeySequence &sequence) {
+
+    // if (debug) qDebug() << InputContextName << __PRETTY_FUNCTION__ << "action" << action;
+
+    // NOTE: currently not trying to trigger action directly
```

Comment:
It's not clear what this means to me. If you think it's needed to have a comment, be very specific with what you mean

Reply:
Hmmm...I myself don't fully understand it...probably one of the things in my old branch that isn't applicable anymore. I will remove it.

Reply:
I've deleted the comment in the latest commit.